daemons/gptp/common/ieee1588port.cpp

```diff
@@ -145,16 +171,9 @@ bool IEEE1588Port::init_port(int delay[4])
 	this->net_iface = net_iface;
 	this->net_iface->getLinkLayerAddress(&local_addr);
 	clock->setClockIdentity(&local_addr);
+	memcpy(this->link_delay, delay, 4);
```

Comment:
delay is an array of 4 ints, and it passed as parameter from daemon_cl (both win/lin). The intent here is not to repeat the code. I wanted to call the _hw_timestamper->HWTimestamper_init again when the link would come up, so I wrapped that in another function and created a new private variable that would contain the four integers from the phy delay info.